README.md

```diff
@@ -429,46 +430,35 @@ Valid architecture values: `x86`, `x86_64`, `armeabi-v7a` `arm64-v8a`, by defaul
 ```bash
 adb install .tmp/src/android-build/build/outputs/apk/debug/android-build-debug.apk
 ```
+#### Building using Conda on Windows
 
-### How to build from source code for Windows
+We provide a Conda env for easy Setup. 
+Prerequisites: 
+- Have [miniconda installed](https://repo.anaconda.com/miniconda/Miniconda3-py310_23.1.0-1-Windows-x86_64.exe).
+- Called conda init with powershell.
+- You need to enable [powershell scripts.](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-7.3)
+  - i.e to enable scripting in your current terminal: ```Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope Process ```
```

Comment:
Nit: Since this is a use case example for the previous bullet point, I'd group them together.